sig-architecture/production-readiness.md

```diff
@@ -23,8 +23,12 @@ aspects of the process.
     plane?
   - Will enabling / disabling the feature require downtime or reprovisioning
     of a node?
+    - What controls exist to to avoid accidental deletion or misconfiguration of
+    this feature?
   - What happens if a cluster with this feature enabled is rolled back? What
     happens if it is subsequently upgraded again?
+  - If the feature needs to be re-enabled, how quickly can you recreate the 
```

Comment:
Maybe also asking how it could be done? Or if the config is lost on rollback? Just asking how long doesn't tell an operator how to do it. I'd like the answers to serve as a guide for the operator.